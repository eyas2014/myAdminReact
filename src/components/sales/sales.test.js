import React from 'react';
import Sales from './sales';
import enzyme from 'enzyme';

const wrapper=enzyme.shallow(<Sales></Sales>);

it ('should match the snapshot', ()=>{
	expect(wrapper).toMatchSnapshot();
})
