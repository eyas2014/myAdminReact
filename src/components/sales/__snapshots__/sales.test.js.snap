// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Sales />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="title"
        >
          <i
            className="fa fa-inbox"
          />
           Sales
        </div>,
        <ul
          className="nav nav-tabs justify-content-end"
        >
          <li
            className="nav-item"
          >
            <a
              aria-expanded="false"
              className="nav-link"
              data-toggle="tab"
              href="#revenue-chart"
            >
              Area
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              aria-expanded="true"
              className="nav-link active"
              data-toggle="tab"
              href="#sales-chart"
            >
              Donut
            </a>
          </li>
        </ul>,
        <div
          className="tab-content no-padding"
        />,
      ],
      "className": "sales",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <i
              className="fa fa-inbox"
            />,
            " Sales",
          ],
          "className": "title",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "fa fa-inbox",
            },
            "ref": null,
            "rendered": null,
            "type": "i",
          },
          " Sales",
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <li
              className="nav-item"
            >
              <a
                aria-expanded="false"
                className="nav-link"
                data-toggle="tab"
                href="#revenue-chart"
              >
                Area
              </a>
            </li>,
            <li
              className="nav-item"
            >
              <a
                aria-expanded="true"
                className="nav-link active"
                data-toggle="tab"
                href="#sales-chart"
              >
                Donut
              </a>
            </li>,
          ],
          "className": "nav nav-tabs justify-content-end",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <a
                aria-expanded="false"
                className="nav-link"
                data-toggle="tab"
                href="#revenue-chart"
              >
                Area
              </a>,
              "className": "nav-item",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "aria-expanded": "false",
                "children": "Area",
                "className": "nav-link",
                "data-toggle": "tab",
                "href": "#revenue-chart",
              },
              "ref": null,
              "rendered": "Area",
              "type": "a",
            },
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <a
                aria-expanded="true"
                className="nav-link active"
                data-toggle="tab"
                href="#sales-chart"
              >
                Donut
              </a>,
              "className": "nav-item",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "aria-expanded": "true",
                "children": "Donut",
                "className": "nav-link active",
                "data-toggle": "tab",
                "href": "#sales-chart",
              },
              "ref": null,
              "rendered": "Donut",
              "type": "a",
            },
            "type": "li",
          },
        ],
        "type": "ul",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "tab-content no-padding",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="title"
          >
            <i
              className="fa fa-inbox"
            />
             Sales
          </div>,
          <ul
            className="nav nav-tabs justify-content-end"
          >
            <li
              className="nav-item"
            >
              <a
                aria-expanded="false"
                className="nav-link"
                data-toggle="tab"
                href="#revenue-chart"
              >
                Area
              </a>
            </li>
            <li
              className="nav-item"
            >
              <a
                aria-expanded="true"
                className="nav-link active"
                data-toggle="tab"
                href="#sales-chart"
              >
                Donut
              </a>
            </li>
          </ul>,
          <div
            className="tab-content no-padding"
          />,
        ],
        "className": "sales",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <i
                className="fa fa-inbox"
              />,
              " Sales",
            ],
            "className": "title",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fa fa-inbox",
              },
              "ref": null,
              "rendered": null,
              "type": "i",
            },
            " Sales",
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <li
                className="nav-item"
              >
                <a
                  aria-expanded="false"
                  className="nav-link"
                  data-toggle="tab"
                  href="#revenue-chart"
                >
                  Area
                </a>
              </li>,
              <li
                className="nav-item"
              >
                <a
                  aria-expanded="true"
                  className="nav-link active"
                  data-toggle="tab"
                  href="#sales-chart"
                >
                  Donut
                </a>
              </li>,
            ],
            "className": "nav nav-tabs justify-content-end",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  aria-expanded="false"
                  className="nav-link"
                  data-toggle="tab"
                  href="#revenue-chart"
                >
                  Area
                </a>,
                "className": "nav-item",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "aria-expanded": "false",
                  "children": "Area",
                  "className": "nav-link",
                  "data-toggle": "tab",
                  "href": "#revenue-chart",
                },
                "ref": null,
                "rendered": "Area",
                "type": "a",
              },
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  aria-expanded="true"
                  className="nav-link active"
                  data-toggle="tab"
                  href="#sales-chart"
                >
                  Donut
                </a>,
                "className": "nav-item",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "aria-expanded": "true",
                  "children": "Donut",
                  "className": "nav-link active",
                  "data-toggle": "tab",
                  "href": "#sales-chart",
                },
                "ref": null,
                "rendered": "Donut",
                "type": "a",
              },
              "type": "li",
            },
          ],
          "type": "ul",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "tab-content no-padding",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
